scripts/kwil_binaries.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+download_binaries() {
+    local ARCH=$(uname -m)
+    local OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+
+    # Determine the architecture
+    if [[ "$ARCH" == "x86_64" ]]; then
+        ARCH="amd64"
+    elif [[ "$ARCH" == "aarch64" || "$ARCH" == "arm64" ]]; then
+        ARCH="arm64"
+    else
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+    fi
+
+    # Determine the operating system
+    if [[ "$OS" == "linux" ]]; then
+        OS="linux"
+    elif [[ "$OS" == "darwin" ]]; then
+        OS="darwin"
+    else
+        echo "Unsupported operating system: $OS"
+        exit 1
+    fi
+
+    # Set the URL for the binary
+    URL="https://github.com/kwilteam/kwil-db/releases/download/v0.8.1/kwil-db_0.8.1_${OS}_${ARCH}.tar.gz"
```

Comment:
Could you update this to Kwil v0.8.4?

https://github.com/kwilteam/kwil-db/releases/tag/v0.8.4